framework/src/variables/MooseVariableFV.C

```diff
@@ -372,10 +374,9 @@ template <typename OutputType>
 OutputType
 MooseVariableFV<OutputType>::getValue(const Elem * elem) const
 {
-  std::vector<dof_id_type> dof_indices;
-  this->_dof_map.dof_indices(elem, dof_indices, _var_num);
-  mooseAssert(dof_indices.size() == 1, "Wrong size for dof indices");
-  OutputType value = (*this->_sys.currentSolution())(dof_indices[0]);
+  Moose::initDofIndices(const_cast<MooseVariableFV<OutputType> &>(*this), *elem);
```

Comment:
This is a very nice catch, especially when we have gradient evaluations where we use the value so many times!